src/tests/orca-jedi/test_interpolator.cc

```diff
@@ -114,7 +114,7 @@ CASE("test basic interpolator") {
   }
   SECTION("test interpolator.apply multiple levels") {
     std::vector<double> vals(nlevs*nlocs);
-    std::vector<bool> mask(nlevs*nlocs);
+    std::vector<bool> mask(nlocs, true);
```

Comment:
Its not clear here why the mask is only nlocs long, it makes it look as though it is only one mask per level in the data which isn't true since this is a time mask? Can we give some different names to some things?

Reply:
I don't think it's a time mask in that way. I think it should be size nlocs and this is then updated at every time step. I.e. it first sets the mask to true to each observation that is closest to the time of the first model step and then runs the interpolation for that model time step. Then it goes on to the next model time step and updates the mask variable so that it is true for the observations that are closest to that time point and false for the others and so on. This is just what I managed to work out from the code, so it is possible that I misunderstood something.

Reply:
Thanks sorry I just misunderstood!